src/main/java/no/digipost/api/datatypes/DataTypeIdentifier.java

```diff
@@ -49,6 +61,10 @@
         this.dataType = dataType;
         this.shortName = shortName;
         this.example = example;
+        ComplementedBy complementedBy = this.getDataType().getAnnotation(ComplementedBy.class);
```

Comment:
```suggestion
        complementables = Optional.ofNullable(getDataType().getAnnotation(ComplementedBy.class))
            .map(ComplementedBy::value)
            .map(Stream::of)
            .orElseGet(Stream::empty)
            .collect(collectingAndThen(toSet(), Collections::unmodifiableSet));
```
Mulig ikke _absolutt alt_ over ikke kompilerer, men tror det skal være noe sånt for å slippe å dille med mutable sets som du må wrappe ting i liste for å add'et de, etc.